src/communication/network_histogram.hpp

```diff
@@ -0,0 +1,87 @@
+/*
+ *
+ * Copyright 2016 CUBRID Corporation
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ *
+ */
+
+/*
+ * network_histogram..hpp - Add instrumentations to the client side to get histogram of network requests
```

Comment:
There are 2 dots, is it a typo?
```suggestion
 * network_histogram.hpp - Add instrumentations to the client side to get histogram of network requests
```

Reply:
thank you